opendistro-elasticsearch-alerting.release-notes.md

```diff
@@ -1,18 +1,23 @@
-## Version 1.2.0 (Current)
+## Version 1.2.0.1 (Current)
+
+### Bug fixes
+  * Updated execute API to keep thread context - [PR #90](https://github.com/opendistro-for-elasticsearch/alerting/pull/90)
```

Comment:
Master branch has more changes since ODFE 1.2 release. Can you add those in this release-note?